src/System.Security.Cryptography.Algorithms/pkg/System.Security.Cryptography.Algorithms.pkgproj

```diff
@@ -5,8 +5,11 @@
     <ProjectReference Include="..\ref\4.0\System.Security.Cryptography.Algorithms.csproj">
       <SupportedFramework>net46</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\ref\4.1\System.Security.Cryptography.Algorithms.csproj">
+      <SupportedFramework>net461;netcore50;netcoreapp1.0</SupportedFramework>
+    </ProjectReference>
     <ProjectReference Include="..\ref\System.Security.Cryptography.Algorithms.csproj">
-      <SupportedFramework>net461;netcore50;netstandardapp1.5</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.0</SupportedFramework>
     </ProjectReference>
```

Comment:
@ericstj please verify the pkgproj. Note sure if I can reference netcoreapp1.0 in two location as shown. Will need to re-build once your changes are in. Thanks